meteor/lib/collections/Parts.ts

```diff
@@ -133,11 +135,7 @@ export class Part implements DBPart {
 			? [
 					{
 						type: NoteType.ERROR,
-						message: {
-							key: `${this.invalidReason.title}${
-								this.invalidReason.description ? `: ${+this.invalidReason.description}` : ''
-							}`,
-						},
+						message: this.invalidReason.message,
```

Comment:
I expect this will need a namespace to be defined

Reply:
Core should now add the namespace for this message on ingest in 30e1ff1a21114b88c2477b2872b74eee3af79256